website/docs/getting_started.html.markdown

```diff
@@ -0,0 +1,224 @@
+---
+layout: "google"
+page_title: "Getting Started with the Google provider"
+sidebar_current: "docs-google-provider-getting-started"
+description: |-
+  Getting started with the Google Cloud Platform provider
+---
+
+# Getting Started with the Google Provider
+
+## Before you begin
+
+* You'll want to have created an account and project in the [Google Cloud Console](https://console.cloud.google.com/)
+and set up billing on that project. Any examples in this guide will be part of
+the [GCP "always free" tier](https://cloud.google.com/free/).
+* [Install Terraform](https://www.terraform.io/intro/getting-started/install.html)
+and read the Terraform getting started guide that follows. This guide will
+assume basic proficiency with Terraform - it is an introduction to the Google
+provider.
+
+## Configuring the Provider
+
+To configure the provider, first create a Terraform config file. Inside, you'll
+want to include the following configuration:
+
+```hcl
+provider "google" {
+  project = "{{YOUR GCP PROJECT}}"
+  region  = "us-central1"
+  zone    = "us-central1-c"
+}
+```
+
+* The `project` field should include your personal project id. The `project`
+indicates the default GCP project all of your resources will be created in.
+Most Terraform resources will have a `project` field.
+* The`region` and `zone` are [locations](https://cloud.google.com/compute/docs/regions-zones/global-regional-zonal-resources)
+for your resources to be created in.
+    * The `region` will be used to choose the default location for regional
+    resources. Regional resources are spread across several zones.
+    * The `zone` will be used to choose the default location for zonal resources.
+    Zonal resources exist in a single zone. All zones are a part of a region.
+
+Not all resources require a location. Some GCP resources are global and are
+automatically spread across all of GCP.
+
+-> Want to try out another location? Check out the [list of available regions and zones](https://cloud.google.com/compute/docs/regions-zones/#available).
+Instances created in zones outside the US are not part of the always free tier
+and could incur charges.
+
+## Creating a VM instance
+A [Google Compute Engine VM instance](https://cloud.google.com/compute/docs/instances/) is
+named `google_compute_instance` in Terraform. The `google` part of the name
+identifies the provider for Terraform, `compute` indicates the GCP product
+family, and `instance` is the resource name.
+
+Google provider resources will generally, although not always, be named after
```

Comment:
Maybe mention something about that name also being used for gcloud, since people will probably be familiar with that and not with the rest api